docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,35 @@ metadata:
     tenant-name: {TENANT_NAME}
     hyperscaler-type: "gcp_cf-sa30"
 ```
+
+## HAP Configuration
+
+<!-- selection algorithm -->
+When selecting a Secret for a given hyperscaler, KEB evaluates a set of rules to determine what secret bindings it should query for a given plan and region that SKR is provisioned in. The search looks for a secret bindings with `hyperscaler-type` label only (always used in the search) or additionally with one of or both `euAccess: true` and `shared: true` labels. 
+
+`hyperscaler-type` label is the one that does contain region information in the format `hyperscaler_type: <HYPERSCALER_NAME>[_<PLATFORM_REGION>][_<CLUSTER_REGION>]`, where both `_<PLATFORM_REGION>` and `_<CLUSTER_REGION>` are optional.
```

Comment:
Could we stick to `hyperscaler region` (HYPERSCALER_REGION accordingly) as in #1520

Reply:
I want something shorter. HAP_REGION?